src/clang-c-frontend/clang_c_convert.cpp

```diff
@@ -2477,6 +2517,16 @@ bool clang_c_convertert::get_cast_expr(
 
   case clang::CK_VectorSplat:
     break;
+
+#ifdef ESBMC_CHERI_CLANG
+  case clang::CK_PointerToCHERICapability:
+    /* An explicit __cheri_tocap means this value might be tagged. */
+  case clang::CK_CHERICapabilityToPointer:
+    /* both should not be generated in purecap mode */
+    // assert(0);
```

Comment:
Do you know if we need to keep this comment code here?

Reply:
Removed.